pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/kubelet.go

```diff
@@ -184,6 +187,10 @@ func KubeletConfigParametersFromCoreV1beta1KubeletConfig(kubeletConfig *gardenco
 				out.EvictionSoftGracePeriod[NodeFSInodesFree] = val.Duration.String()
 			}
 		}
+
+		if val := kubeletConfig.ContainerLogMaxSize; val != nil {
+			out.ContainerLogMaxSize = pointer.String(val.String())
+		}
```

Comment:
Can you move to L86 to have both ContainerLogMaxFiles and ContainerLogMaxSize next to each other?

Reply:
ok